src/tools/rust-demangler/main.rs

```diff
@@ -0,0 +1,39 @@
+//! Demangles rustc mangled names.
```

Comment:
The rustfilt tool also does something like this, though I'm not sure what our policy is on using tools outside the rust-lang org.

Reply:
Good to point out. 

If `rustfilt` was already part of the `rust-lang` distribution, I probably would have used it, but since it's not, the `rust-demangler` tool is a closer fit for the use case.

It looks like `rustfilt` has some nice features, but more than we need to support the expected behavior for `llvm-cov --Xdemangler=tool_path`.

That said, if there's a preference to use it, and no barriers, I'm fine with that too.

Reply:
No preference from me, really.